ui/src/components/tag/AddTagForm.js

```diff
@@ -0,0 +1,223 @@
+            if (tagValues.indexOf(names[i]) === -1) {
+                tagValues.push(names[i]);
+            }
+        }
+
+        this.setState(
+            {
+                tagValues,
+                newTagValue: '',
+            },
+            () => this.props.onUpdate(this.state.tagName, this.state.tagValues)
+        );
+    }
+
+    removeValue(idx) {
+        let tagValues = this.state.tagValues;
+        tagValues.splice(idx, 1);
+        this.setState({ tagValues }, () =>
+            this.props.onUpdate(this.state.tagName, this.state.tagValues)
+        );
+    }
+
+    render() {
+        let tagValueChanged = this.inputChanged.bind(this, 'newTagValue');
+        let tagNameChanged = this.inputChanged.bind(this, 'tagName');
+
+        let tagValues = this.state.tagValues
+            ? this.state.tagValues.map((val, idx) => {
+                  let remove = this.removeValue.bind(this, idx);
+                  return (
+                      <StyledTagColor key={val} onClickRemove={remove}>
+                          <StyledAnchor style={StyledAnchorActiveInline}>
+                              {' '}
+                              {val}{' '}
+                          </StyledAnchor>
+                      </StyledTagColor>
+                  );
+              })
+            : '';
+
+        return (
+            <SectionsDiv autoComplete={'off'} data-testid='add-tag-form'>
+                <SectionDiv>
+                    <StyledInputLabel htmlFor='tag-name'>
+                        Tag Name
+                    </StyledInputLabel>
+                    <ContentDiv>
+                        <StyledInput
+                            id='tag-name'
+                            name='tag-name'
+                            placeholder='Enter New Tag Name'
+                            value={this.state.tagName}
+                            onChange={tagNameChanged}
+                        />
+                    </ContentDiv>
+                </SectionDiv>
+                <SectionDiv>
+                    <StyledInputLabel htmlFor='description'>
+                        Tag Value(s)
+                    </StyledInputLabel>
+                    <ContentDiv>
+                        <StyledInput
+                            id='tag-val'
+                            name='tag-val'
+                            placeholder='Enter New Tag Value'
+                            value={this.state.newTagValue}
+                            onChange={tagValueChanged}
+                        />
+                    </ContentDiv>
+                </SectionDiv>
+                <SectionDiv>
+                    <StyledInputLabel />
+                    <ButtonDiv>
+                        <StyledButton
+                            secondary
+                            size={'small'}
+                            onClick={this.addTagValue}
+                        >
+                            Add
```

Comment:
let's have the add button next to the tag value field instead of on the next line

Reply:
This is same pattern as add role member